src/script/hydra-eval-jobset

```diff
@@ -348,7 +348,7 @@ sub evalJobs {
                 inputsToArgs($inputInfo));
     }
 
-    push @cmd, "--no-allow-import-from-derivation";
+    push @cmd, "--no-allow-import-from-derivation" if $config->{disallow_import_from_derivation} // "false" ne "true";
```

Comment:
Maybe invert this to `allow_import_from_derivation` to make it the same as the nix.conf option?

Reply:
Good point @edolstra. I updated the PR, please take a look.